code/modules/materials/definitions/materials_metal.dm

```diff
@@ -216,12 +216,12 @@
 /material/plasteel/ocp
 	name = MATERIAL_OSMIUM_CARBIDE_PLASTEEL
 	stack_type = /obj/item/stack/material/ocp
-	integrity = 200
+	integrity = 8000
```

Comment:
the integrity jump for this (and plastitanium above) is huge; why?

Reply:
Put it on par with the rest (steel/titanium)